index.html

```diff
@@ -231,5 +231,22 @@ <h3>LET'S CONNECT</h3>
               </ul>
              <div class="divider"></div>
        </section>
+       <footer class="contact-form">
+        <form class="contact-info" action="https://formspree.io/f/mwkyenjw" method="POST">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <div class="input-container">
+                <input type="text" name="Name" class="contact-input" maxlength="30" required placeholder="Write your name">
+                <input type="email" name="_replyTo" class="contact-input" placeholder="Write your email" required>
+                <textarea name="message" class="contact-textarea"  cols="30" rows="10" required maxlength="500" placeholder="Write your message here"></textarea>
+                <div class="works-flex-item get-it">
+                 <div class="btn-get no-padding">
+                   <button type="submit" class="btn-get-it-touch">Get It Touch</button>
```

Comment:
- [ ] [OPTIONAL] I know this is an error but please update `Get It Touch` to `Get In Touch` 😊😊😊